app/models/feedback.rb

```diff
@@ -0,0 +1,29 @@
+class Feedback < ActiveRecord::Base
+  belongs_to :user
+
+  validate :rate_limit_per_5_seconds, on: :create
+  validate :rate_limit_per_day, on: :create
+
+  after_create :send_feedback
+
+  scope :last_5_seconds, -> { where("date_sub(now(), interval 5 second) <= created_at") }
+  scope :last_24_hours, -> { where("date_sub(now(), interval 24 hour) <= created_at") }
+
+  private
+
+  def rate_limit_per_5_seconds
+    if Feedback.where(remote_ip: self.remote_ip).last_5_seconds.exists?
+      errors.add(:base, :rate_limit_per_5_seconds)
+    end
+  end
+
+  def rate_limit_per_day
+    if Feedback.where(remote_ip: self.remote_ip).last_24_hours.count >= 10
```

Comment:
Feels like a very low limit, given how some IP addresses are shared by many machines.

How about we:
* put the limit constant on a config
* report to NewRelic whenever the limit is reached